hardhat.config.js

```diff
@@ -0,0 +1,78 @@
+const { utils, ethers } = require("ethers");
+const fs = require("fs");
+const chalk = require("chalk");
+
+require("@nomiclabs/hardhat-waffle");
+
+const defaultNetwork = "localhost"; // "hardhat" for tests
+const INFURA_API_KEY = "ffffffffffffffffffffffffffffffff";
+const TESTNET_ACCOUNT_PRIVATE_KEY = "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff";
```

Comment:
Add the `INFURA_API_KEY` and `TESTNET_ACCOUNT_PRIVATE_KEY`  in a .env file and access them here if you are using them